pages/apply.html

```diff
@@ -1,210 +1,323 @@
+                First Meeting
+                <br />
+                Oct. 18
+              </p>
+            </div>
           </div>
         </div>
-      </div>
-
-      <div class="section">
-        <h5>MODEL</h5>
-        <div class="row">
-          <div class="col-3-xl ml-lg-3 my-1">
-            <p>
-              Audition Training
-              <br> Sept.11-12 <span id="atSign">@</span> 10PM-12AM EST
-              <br> <a href="https://cmu.zoom.us/j/94374218710?pwd=clI3Z2dVU0Z2Qm9MdkxMQVR3VERqZz09" target="_blank" rel="noopener noreferrer">Zoom</a> / Pass: 662304
-              <br> + <a href="https://calendar.google.com/event?action=TEMPLATE&tmeid=NWZoNmY5a2JyZmdyOWszYmwyNWNkMHJoc2sgcHFpbkBhbmRyZXcuY211LmVkdQ&tmsrc=pqin%40andrew.cmu.edu">Google Calendar</a>
-            </p>
-          </div>
-          <div class="col-3-xl ml-lg-3 my-1">
-            <p>
-              Auditions
-              <br> 9.18 <span id="atSign">@</span> 3-7PM
-              <br> + <a href="https://calendar.google.com/event?action=TEMPLATE&tmeid=NTQ2NjEybXBxNzY5M3Y3NG9hcTdoZ2ZkZjQgY18zY3EyYWNrN241ZGdpMWg5djlpbjc4YW91b0Bn&tmsrc=c_3cq2ack7n5dgi1h9v9in78aouo%40group.calendar.google.com">Google Calendar</a>
 
-              <br>9.19 <span id="atSign">@</span> 2-6PM
-              <br> + <a href="https://calendar.google.com/event?action=TEMPLATE&tmeid=NmYzODVwZnYzamlnbGttbjdzc2dwNWQxNmkgY18zY3EyYWNrN241ZGdpMWg5djlpbjc4YW91b0Bn&tmsrc=c_3cq2ack7n5dgi1h9v9in78aouo%40group.calendar.google.com">Google Calendar</a>
+        <div class="section">
+          <h5>MODEL</h5>
+          <div class="row">
+            <div class="col-3-xl ml-lg-3 my-1">
+              <p>
+                Pre-Audition Training (optional)
+                <br />
+                Sept. 27 & 29 <span id="atSign">@</span> 10PM-12AM EST <br />
+                Location: Simmons A <br /><br /><i>Bring heels</i><br />
+              </p>
+            </div>
+            <div class="col-3-xl ml-lg-3 my-1">
+              <p>
+                Auditions
+                <br />
+                TBD<br /><br />
+                <i>Bring 4+ inch heels and wear black tight fitting clothes</i>
 
-              <br>9.20 <span id="atSign">@</span> 1-3PM
-              <br> + <a href="https://calendar.google.com/event?action=TEMPLATE&tmeid=NWlwNzhxczNpZmJmOWpvZnBzZmMzbGwzNTQgY18zY3EyYWNrN241ZGdpMWg5djlpbjc4YW91b0Bn&tmsrc=c_3cq2ack7n5dgi1h9v9in78aouo%40group.calendar.google.com">Google Calendar</a>
-
-              <br><a href="https://cmu.zoom.us/j/96709488615?pwd=Q3FlUURwUGFPRnVOZ0tWMEduMTcyQT09" target="_blank" rel="noopener noreferrer">Zoom</a> / Pass: 662304
-            </p>
+                <br />
+                <!-- <a
+                  href="https://cmu.zoom.us/j/96709488615?pwd=Q3FlUURwUGFPRnVOZ0tWMEduMTcyQT09"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  >Zoom</a
+                > -->
+                <!-- / Pass: 662304 -->
```

Comment:
Again, no need to comment this out, just delete it.

Also reminder that comments still appear on page sources, so people can find this if they inspect element